benchmark/testsystems/bookkeepers.py

```diff
@@ -2,7 +2,7 @@
 import os
 
 import numpy as np
-from openmmtools.integrators import GHMCIntegrator, GradientDescentMinimizationIntegrator, VVVRIntegrator
+from benchmark.integrators.kyle.xchmc import XCGHMCIntegrator
```

Comment:
Needs to be in a submodule of `benchmark.integrators` at least, otherwise would introduce name collisions. Can rename `kyle` to `xc` or something if you'd prefer?

Propose to de-prioritize merging this into `openmmtools.integrators` until it's more thoroughly tested, to minimize the amount of integrator-related user support we need to provide until the paper is submitted.